tests/gtests/test_SW_Markov.cc

```diff
@@ -195,20 +191,18 @@ namespace {
       "Bad covariance matrix"
     );
 
-    // Reset to previous global state
-    // Reset_SOILWAT2_after_UnitTest();
     SW_MKV_deconstruct(&SW_All.Markov);
   }
 
 
   // Test correcting daily temperatures for wet/dry days
-  TEST(WGTest, WetDryTemperatureCorrection) {
+  TEST_F(AllTest, WetDryTemperatureCorrection) {

```

Comment:
Please change `TEST_F(AllTest, WetDryTemperatureCorrection)` to something like `TEST_F(AllTest, WeatherGeneratorWetDryTemperatureCorrection)`